employee/models.py

```diff
@@ -29,39 +27,42 @@ class Employee(models.Model):
     dob = models.DateField(verbose_name='Date of Birth')
     gender = models.IntegerField(choices=GENDER)
     email = models.EmailField(max_length=255, unique=True)
-    phone_number = models.PositiveIntegerField()
+    phone_number = models.CharField(max_length=16, validators=[
+        RegexValidator(
+            regex=r'^\+977\s-?\d{10}$',
+            message="Phone number must be entered in the format '+977 9999999999'."
+        ),
+    ],)
     address = models.CharField(max_length=255)
     pan_no = models.CharField(max_length=255, unique=True)
     citizenship_no = models.CharField(max_length=255, unique=True)
-    designation = models.ForeignKey(
-        Designation, on_delete=models.DO_NOTHING)
+    designation = models.ForeignKey(Designation, on_delete=models.DO_NOTHING)
     join_date = models.DateField()
-    picture = models.ImageField(upload_to='pictures/')
-    pan_no_document = models.ImageField(upload_to='pictures/')
-    cititzen_document = models.ImageField(upload_to='pictures/')
+    picture = models.ImageField(upload_to='images/pictures/')
+    pan_no_document = models.ImageField(upload_to='images/pans/')
+    cititzen_document = models.ImageField(upload_to='images/citizenships/')
     status = models.IntegerField(choices=STATUSES)
     createdBy = models.ForeignKey(
-        User, on_delete=models.DO_NOTHING, related_name="createdBy")
+        User, on_delete=models.DO_NOTHING, related_name="createdBy"
```

Comment:
Lets fix related name
```suggestion
        User, on_delete=models.DO_NOTHING, related_name="created_designations"
        
```

such as, I can do
- `designation.created_by`, to get user
- `user.created_designations.all()`, to get list of designations created by the user.

Reply:
Changes made:
created_by = models.ForeignKey(
        User, on_delete=models.PROTECT, related_name="created_by"
    )
    updated_by = models.ForeignKey(
        User, on_delete=models.PROTECT, related_name="updated_by"
    )